addons/web/static/src/views/basic_relational_model.js

```diff
@@ -562,6 +562,8 @@ export class Record extends DataPoint {
      * @param {boolean} [options.useSaveErrorDialog=false] displays a custom
      *  dialog and await the response from this dialog when an error is
      *  returned by the server.
+     * @param {boolean} [options.throwOnError=false] throws the saving error if
```

Comment:
I'm wondering if we should not put both commits together. as this one has no real value without the other and the other one is totally dependent on this one.

Reply:
The idea was to isolate the framework change from our use for reviewers (with links). If you confirm, I'll merge them.

Reply:
then I confirm ! :)